385/Assets/Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Assets.Scripts;
+
+public class PlayerCamera : MonoBehaviour {
+    // player object camera is attached to
+    public GameObject player;
+
+    // offset of camera from player
+    private Vector3 offset;
+
+    // Static position of camera
+    private Vector3 initPos;
+
+    // camera that follows player
+    public Camera playerCam;
+
+    // checks Camera1 to change camera settings
+    public string CameraAxis = "Camera1";
+
+    // used to check is camera button is pushed
+    private AxisButton CameraButton;
+
+    public bool trackPlayer = false;
+
+    public float moveSpeed = 1;
+
+    public float distance = 2, X; // controls distance of camera and 
+                                  // its X axis adjustment
+
+    private bool camMoving = false;
+
+    public RopeSystem RopeSystem;
+
+    // gets offset for camera and starts camera in static view
+    void Start () {
+        float Y = distance / 2; // Y axis adjustment for changing the camera distance
+        Vector3 temp = new Vector3(-X, Y, distance);  // stores adjustments
```

Comment:
This value should be determined each time that the camera zooms in and out instead of only once at start, so that the controls in the unity editor are able to modify the values in testing. Makes things easier to debug.

Reply:
offset now updated each frame to show changes to distance, X and Y offset during testing

Reply:
Why not Z?